release-notes.txt

```diff
@@ -1,3 +1,491 @@
++ drivers/st77xx: introduce rotation defines (#19919)
+* boards/adafruit-itsybitsy-m4: configure littleFS on external flash (#19355)
+* boards/nucleo64: fix SPI Arduino mapping for most boards (#19935)
+* boards/sipeed_longan_nano: separate board definition for Sipeed
+  Longan Nano TFT (#19824)
+* boards/stm32f469i-disco: enable FMC with SDRAM support (#19910)
+* boards: complete SD Card MTD definition for several bords (#19914)
+* sys: PSA Crypto API implementation (#18547)
+
+CPU
+---
+
+* cpu/atmega_common: hook up BAT LOW irq to power bus (#19822)
+* cpu/riscv_common: remove picolibc from blacklisting in CI (#19862)
+* cpu/stm32/periph/eth: Disable hardware checksums (#19952)
+* cpu/stm32: bump cmsis packages version (#19904)
+* cpu/stm32: fix ld script for SRAM4 (#19842)
+* dist/tools/esptools: upgrade ESP32x toolchains to GCC version 12.2 (#19452)
+* sys/psa_crypto: Ed25519 (EdDSA) support (#19954)
+* tree-wide: mixed box of compilation fixes with clang (#19634)
+
+Device Drivers
+--------------
+
++ drivers/lcd: add MCU 8080 16-bit parallel mode support (#19937)
++ drivers/lcd: add MCU-driven low-level parallel interface (#19941)
++ drivers/lcd: support MCU 8080 8-bit parallel mode (#19915)
++ drivers/touch_dev_gestures: add gesture recognition for touch devices (#19884)
+* drivers/ft5x06: use a pointer to config parameters instead of copying
+  them (#19866)
+* drivers/sdmmc: store SDMMC device descriptor references in XFA (#19899)
+* drivers/stmpe811: changes for interrupt-driven touch handling and
+  gesture recognition (#19885)
+
+Documentation
+-------------
+
++ drivers/periph: Add documentation on thread safety and initialization (#19794)
+* boards/sltb009a: complete and fix documentation (#19888)
+* doc: fix references and inches unit (#19948)
+
+Build System / Tooling
+----------------------
+
++ compile_and_test_for_boards: Add no-compile flag (#19817)
++ dist/testbed-support: Add openmote board [backport 2023.10] (#19984)
+* dist/tools/jlink: fix DBG_PID assignment (#19960)
+* dist/tools/usb-serial: call ttys.py with its path (#19823)
+* make: COMPILE_COMMANDS_PATH adapt for external apps (#19869)
+
+
+Examples
+--------
+
++ examples/gcoap: add saml11-xpro to CI boards with insufficient memory (#19933)
+
+Testing
+-------
+
++ tests/drivers/touch_dev: allow to test a touch device in polling mode (#19882)
++ tests/pkg/lwip: Add README.md (#19949)
+* .github/test-on-iotlab: prefer Toulouse site for dwm1001 board (#19950)
+* .github: drop test-on-ryot workflow (#19847)
+* clang floating point handling fix (#19852)
+* gh-actions: remove reporting release tests to Matrix (#19879)
+* tests/gcoap_fileserver: only enable test with GCC (#19870)
+* tests/net/gcoap_fileserver: disable test on CI (#19898)
+* tests/net/gcoap_fileserver: Fix failing nightlies (#19856)
+* tests/pkg/relic: skip CI testing with samr21-xpro and llvm toolchain (#19902)
+
+API Changes
+-----------
+
+* drivers/mtd: use XFA for pointers to defined MTDs (#19465)
```

Comment:
Uh this still made it into the release? I thought it was merged after the hard feature freeze.
Anyway, you then probably want to add the porting guide from the PR to the release notes.

Reply:
I can double check... I had to do some hacks in the release note generation.

Reply:
It is not in there.  Maybe it was merged just after the freeze and my hard-coded timestamp took it.

Reply:
Uh does the `set-milestones` sub-command of `riot-release-manager` no longer work?